pwndbg/gdblib/disasm/__init__.py

```diff
@@ -149,17 +153,19 @@ def get_disassembler_cached(arch, ptrsize: int, endian, extra=None):
     return cs
 
 
-def get_disassembler(pc):
+def get_disassembler(address):
     if pwndbg.gdblib.arch.current == "armcm":
+        thumb_mode = bool(emulated_arm_mode_cache[address])
+        if thumb_mode is None:
```

Comment:
It can't be `None` here since we did `bool(...)`. I believe this should be fixed @OBarronCS

Reply:
Oop -  I'll remove all the `bool`s here as we can rely on Python seeing `0` as falsy and any other integer as truthy in this context.